scripts/install_obsolete_jsoncpp_1_7_4.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR=$(mktemp -d)
```

Comment:
Why not just `cd` into `$TEMPDIR` to avoid a lot of prefixing?

Reply:
addressed.